src/args.rs

```diff
@@ -37,13 +41,16 @@ pub struct Args {
     ///
     /// This conflicts with the --query option.
     #[arg(short = 'Q', long = "query-file", conflicts_with = "query_text")]
-    pub path_to_query_file: Option<PathBuf>,
+    path_to_query_file: Option<PathBuf>,
 
     /// The source text of a tree-sitter query.
     ///
     /// This conflicts with the --query-file option.
     #[arg(short, long = "query", conflicts_with = "path_to_query_file")]
-    pub query_text: Option<String>,
+    query_text: Option<String>,
+
+    #[clap(skip)]
+    query_per_language: Option<HashMap<SupportedLanguage, Arc<Query>>>,
```

Comment:
This is "tackling a couple birds with one stone" (?), first this is allowing passing in pre-parsed queries, second this is supporting "query-per-language" (vs "just one query")

Query-per-language allows `tree-sitter-lint` to let a single "run" of `tree-sitter-grep` "drive" everything, which seems to make sense (so that we're only doing one project-file-walking)